packages/@okta/vuepress-site/docs/guides/policy-simulation/main/index.md

```diff
@@ -0,0 +1,268 @@
+---
+title: Test your policies with access simulations
+excerpt: Provides documentation on testing your policies using the Policy APIs simulate endpoint
+layout: Guides
+---
+
+<ApiLifecycle access="ea" />
+
+This guide explains how to test your access policies by using the `/simulate` endpoint of the Policy API.
+
+---
+
+**Learning outcomes**
+
+* Understand how to configure and send policy simulation tests using the Policy API.
+* Send and receive an example simulation call.
+
+**What you need**
+
+* [Okta Preview organization](https://developer.okta.com/login)
+* An app to test your access policies
+* The Early Access self-service Access Test Tool feature enabled on your org. See [Early Access](/docs/reference/releases-at-okta/#early-access-ea).
+
+**Sample code**
+
+If you need a simple application for testing, see [Sign users in to your SPA using the redirect model and AuthJS](/docs/guides/auth-js-redirect/main/).
+
+---
+
+## About Policy Simulation
+
+The policy API can be used to simulate real-world user requests to access an application. In the Admin Console, these simulations are run using the Access Testing Tool available from **Reports** > **Access testing tool**. See [Access Testing Tool](https://help.okta.com/en/programs/aps/Content/Topics/betas/closed/access-policy-simulation/test-an-access-scenario.htm). <!--Update this link to alias when ready --> The API endpoint that underpins this tool is also available for developers to simulate policy configurations and to test application access. For full details on the API endpoint, see the [Policy API reference](/docs/reference/api//policy/#access-simulation).
```

Comment:
```suggestion
You can use the Policy API to simulate real-world user requests to access an application. In the Admin Console, these simulations are run using the Access Testing Tool available from **Reports** > **Access testing tool**. See [Access Testing Tool](https://help.okta.com/en/programs/aps/Content/Topics/betas/closed/access-policy-simulation/test-an-access-scenario.htm). <!--Update this link to alias when ready --> The API endpoint that underpins this tool is also available for developers to simulate policy configurations and to test application access. For full details on the API endpoint, see the [Policy API reference](/docs/reference/api/policy/#access-simulation).
```
Is the Policy API not migrated to Redocly?

Reply:
The Policy API is available (but still needs to be reviewed/updated). However, we have added the new endpoint to the reference on Redocly.